### ISSUE TITLE
[Feature]: Insightface More Powerful Version

### ISSUE THREAD

**Issue by drdancm** [Gourieff/sd-webui-reactor#515]
### Feature description

Their github page discusses the newest, most powerful version.

https://github.com/deepinsight/insightface?tab=readme-ov-file#readme

Could Reactor make use of these advances?

**Comment by drdancm** [comment]
How about Inspireface

https://github.com/HyperInspire/InspireFace

**Comment by drdancm** [comment]
FaceFusion has a reference feature, so that you can recognize/identify one of the faces in your target photo and make the FSwap only for that face.  I don't know how well that works, but if it does , could this be done for Reactor?

**Comment by drdancm** [comment]
Here is another face recognition method which might be really useful for Reactor:

https://github.com/deepinsight/insightface/tree/master/recognition/arcface_torch

**Comment by drdancm** [comment]
FaceFusion v 3 actually does a good job detecting Male, Female, Race, as well as having other improvements.

**Comment by Gourieff** [comment]
Yes, they did a great job, I'll take a look when I'll have free time for this
PRs with improvements are always welcome (into the Evolve branch)

**Comment by drdancm** [comment]
You might want to check out FaceFusion 3.1 which runs nicely on Pinoki. It works well with
1. CloseUps  - this is a weakness with current Reactor (Closeups can't deal well with rotated faces)
2. Gender
3. Age
4. Rotated Faces including 90, 180, and 270

Major problem with FaceFusion is lack of batch mode, plus output does not give choice to retain original image name plus tag.
